packages/chevrotain/test/parse/cst_spec.ts

```diff
@@ -128,12 +128,14 @@ function defineTestSuit(recoveryMode) {
             expect(tokenStructuredMatcher(cst.children.myLabel[0], A)).to.be
                 .true
             expect(tokenStructuredMatcher(cst.children.B[0], B)).to.be.true
-            expect(cst.children.myOtherLabel[0].name).to.equal("bamba")
-            expect(cst.children.myOtherLabel[0].recoveredNode).to.be.true
+
+            const bamba = cst.children.myOtherLabel[0] as CstNode
```

Comment:
if we needed to a CAST here does this mean we have a breaking change on our hands?

Reply:
Cast because it is a `CstElement`, so it can either be an `IToken` or a `CstNode`.